### PR TITLE
qalculate, qalculate-gtk: update to 3.14.0, drop noarch.

### PR DESCRIPTION
**Pull request by ashpooljh**

### CODE REVIEW

File: srcpkgs/qalculate/template [hunk 1]
```diff
@@ -13,10 +13,9 @@ maintainer="Artem Zhurikhin <ashpool@xecut.net>"
 license="GPL-2.0-or-later"
 homepage="https://qalculate.github.io"
 distfiles="https://github.com/Qalculate/libqalculate/releases/download/v${version}/libqalculate-${version}.tar.gz"
-checksum=28c46dc8dd975f253627d80c55a6feab7c44d965dce3ceffefc3cafc9a97c457
+checksum=ef422aa54eac7c711ece65dd3a5cbc66370d3e17173465313201897c201e7d3e
 
 libqalculate-data_package() {
-	archs=noarch
```
Comment (ahesford) [inline srcpkgs/qalculate/template:19]: With the dropping of `noarch`, there is no need to have a separate `data` subpackage. Please drop this package entirely, moving its contents to the `libqalculate` subpackage.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/qalculate/template (hunk 1 above)
Comment (ashpooljh) [reply srcpkgs/qalculate/template:19]: I'm not sure how to do this without hurting any existing installations. I added my attempt in a commit to this PR. In my case, the package upgrade went smoothly, but deleting the old `libqalculate-data` package resulted in deletion of the files which are now a part of `libqalculate`.

Comment (ashpooljh) [reply srcpkgs/qalculate/template:19]: Ok, I just found out about `replaces` and it worked. Commit updated.

Comment (ericonr) [reply srcpkgs/qalculate/template:19]: Remember to remove the symlink.

File: srcpkgs/qalculate/template [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Template file for 'qalculate'
 # NOTE: keep this package synchronized with "srcpkgs/qalculate-gtk"
 pkgname=qalculate
-version=3.12.1
-revision=1
+version=3.14.0
+revision=2
```
Comment (ahesford) [inline srcpkgs/qalculate/template:5]: This should still have `revision=1`

**Review by ahesford — CHANGES_REQUESTED** [review]
Please fix the `revision` and squash commits 26c9675 and 206eee8, so there is exactly one commit for the `qalculate` changes and one for the `qalculate-gtk` changes.

**Pull request closed without merging**